external-builds/pytorch/build_prod_wheels.py

```diff
@@ -382,6 +392,8 @@ def main(argv: list[str]):
     install_rocm_p.set_defaults(func=do_install_rocm)
 
     build_p = sub_p.add_parser("build", help="Build pytorch wheels")
+    build_p.add_argument("--find-links", help="Pip find-links to pass to pip install")
```

Comment:
This is actually an argument to the root command, not the build command. So you can remove this, and just add a `--find-links` to your command line before `build`.